### DISCUSSION TITLE
Clicking on a project link from a discussion does not work from the first try

### DISCUSSION THREAD

**Discussion by jadjoubran** [community/community#3742] [Discussions]
Hello,

I think I've stumbled across a bug. If you link to a `project` from a `discussion`, then click on that link, there will be an error:

<img width="472" alt="Screenshot 2021-04-09 at 16 28 22" src="https://user-images.githubusercontent.com/2265232/114195266-9cadc780-9950-11eb-8a4f-47a83f5214e9.png">

Reloading the page fixes it then.

Let me know if you'd like more information as I am able to reproduce it every time.

Thanks!

**Comment by jen-k** [comment]
Hey @jadjoubran 👋 - Thanks for bringing this to our attention! I've been unable to reproduce this myself, would you mind sharing screenshots of the initial steps you took to encounter the bug?

**Reply by jadjoubran** [reply]
Sure thing, I've recorded a screencast! Note, at the end, after reloading, the project works as expected, I just had to trim the video to hide the content of the project.

Maybe this is occurring because the repository is private? Let me know if you need more details!

https://user-images.githubusercontent.com/2265232/114504648-7da08580-9c2f-11eb-96e2-cfeb55e40b65.mov

Thanks!

**Reply by jen-k** [reply]
Thanks so much @jadjoubran! This is super helpful. I'm passing this on to our engineering team to do some additional investigating - will let you know when we've got it figured out!
